test/unit/helpers/apiError.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+var chai = require('chai');
+var expect = require('chai').expect;
+var express = require('express');
+var ApiError = require('../../../helpers/apiError.js');
+
+describe('helpers/apiError', function () {
+
+	var apiError;
+	var validErrorMessage;
+	var validErrorCode;
+
+	beforeEach(function () {
```

Comment:
just `before` sould be ok

Reply:
#812 - 1 c)

Reply:
For me issue you referenced looks like proposal, has this been accepted as a standards?

Reply:
At least it is for me. Feel free to propose another solution if you don't agree.